src/documentation/Intro/_intro.scss

```diff
@@ -43,7 +43,7 @@ $easing: cubic-bezier(0.17, 0.89, 0.32, 1.1); //ease-in-out;
   //background: linear-gradient(90deg, var(--brand-01), 0%, lighten(var(--brand-01), 5) 100%);
   @include breakpoint(bp--md--major) {
     height: 90vh;
-    width: 100vw;
+    width: 99vw;
```

Comment:
What is the effect of the 99vw?

Reply:
The horizontal line on the intro page on windows - Laura showed it the other time

Reply:
Ah, and the 100vw caused the scrollbar?

Reply:
yes